server/api/src/rsFileChksum.cpp

```diff
@@ -23,6 +23,21 @@
 
 using log_api = irods::experimental::log::api;
 
+static int get_read_buffer_size()
+{
+    // get the buffer size from server configuration - default to 1 MiB
+    constexpr int default_read_buffer_size = 1024 * 1024;
+    int read_buffer_size = default_read_buffer_size;
+    try {
+        read_buffer_size = irods::get_advanced_setting<int>("checksum_read_buffer_size_in_bytes");
+    }
+    catch (const irods::exception&) {
+        log_api::debug("{} :: checksum_read_buffer_size_in_bytes not set. Using default of 1 MiB.", __func__);
+        read_buffer_size = default_read_buffer_size;
+    }
+    return read_buffer_size;
```

Comment:
This can be rewritten like so:
```cpp
try {
    return irods::get_advanced_setting<int>("checksum_read_buffer_size_in_bytes");
}
catch (...) {
    log_api::debug("{} :: checksum_read_buffer_size_in_bytes not set. Using default of 1 MiB.", __func__);
    constexpr int default_read_buffer_size = 1024 * 1024;
    return default_read_buffer_size;
}
```

Reply:
I made the change.